### PR TITLE
[e2e]: provide more information upon failure in executing commands in pods

### PR DESCRIPTION
**Pull request by enp0s3**
Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

**What this PR does / why we need it**:
We need more information about remote command execution failures in infra tests
therefore we will print the stdout and stderr in case the assertion fails.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: tests/infra_test.go [hunk 1]
```diff
@@ -585,35 +589,28 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
...
-				&op,
-				"virt-operator",
-				[]string{
-					"cat",
-					"/var/run/secrets/kubernetes.io/serviceaccount/token",
-				})
-			Expect(err).ToNot(HaveOccurred(), "failed executing command on virt-operator")
+			cmd := []string{"cat", "/var/run/secrets/kubernetes.io/serviceaccount/token"}
+			token, stderr, err := tests.ExecuteCommandOnPodV2(virtClient, &op, "virt-operator", cmd)
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(remoteCmdErrMsg, strings.Join(cmd[:], ","), token, stderr, err))
```
Comment (xpivarc) [inline tests/infra_test.go:594]: `strings.Join(cmd[:], ",")` Would ` ` be better?

**Review by xpivarc — COMMENTED** [review]
The PR title is a little bit misleading.

File: tests/infra_test.go (hunk 1 above)
Comment (iholder-redhat) [reply tests/infra_test.go:594]: Why do we need to copy `cmd`?
Wouldn't `strings.Join(cmd, ",")` achieve the same affect without copying?

Comment (enp0s3) [reply tests/infra_test.go]: @xpivarc @iholder-redhat You are correct! I've fixed both of your remarks. Thank you

**Comment by enp0s3** [comment]
> The PR title is a little bit misleading.

@xpivarc I was trying to consolidate these changes in one-line explanation, perhaps didn't go so well :) can you please suggest another title? So I would know how to better summarize for the next time

File: tests/infra_test.go [hunk 2]
```diff
@@ -80,6 +80,10 @@ import (
 	"kubevirt.io/kubevirt/tests/libnet"
 )
 
+const (
+	remoteCmdErrMsg = "failed running `%s` with stdout:\n %v \n stderr:\n %v \n err: \n %v \n"
```
Comment (iholder-redhat) [inline tests/infra_test.go:84]: nit: maybe `remoteCmdErrPattern`?

**Review by iholder-redhat — COMMENTED** [review]
Looks much better! thanks @enp0s3!
/lgtm

**Comment by enp0s3** [comment]
@iholder-redhat Hi, can you PTAL?

**Comment by iholder-redhat** [comment]
/lgtm
Thank you @enp0s3 !

**Pull request merged**